### PR TITLE
Add multi-migrate page

### PR DESCRIPTION
**Pull request by sonochristian**
see: https://discord.com/channels/883303446822924328/883345315799904306/884770419431862282

### CODE REVIEW

File: pages/migrate.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      }
+      const attributes = await Promise.all(attributePromises);
+      const tx = STLV2.multiClaim([...tokenIds], [...attributes]);
+      await handleTx(tx)
+    } catch (err) {
+      console.error(err)
+    }
+  }, [STL, STLV2, handleTx, isReadOnly, tokenIds])
+
+  if (isFallback) {
```
Comment (ethandaya) [inline pages/migrate.tsx]: you have no getStaticProps call so you don't need a fallback handler

File: pages/migrate.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    return <p>
+      loading...
+    </p>
+  }
+
+  return (
+    <Fragment>
+    <Head>
+      <title>Migrate</title>
+      <meta name="description" content={"Migrate your Settlements"}/>
```
Comment (ethandaya) [inline pages/migrate.tsx]: nit, can just be content="..."

File: pages/migrate.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    </Head>
+    <Header/>
+    <main className={styles.main}>
+      <h1>Migrate</h1>
+      <div>
+        {tokenIds.map((_, index) => (
+          <>
```
Comment (ethandaya) [inline pages/migrate.tsx]: nit, this doesn't need a fragment

File: pages/migrate.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            <input style={{marginBottom: 20}} id={index.toString()} key={index} placeholder={'Token ID'} onChange={onChange}/><br></br>
```
Comment (ethandaya) [inline pages/migrate.tsx]: what are the <br>'s doing?

File: pages/migrate.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+          </>
+        ))}
+        <div>
+          <button style={{marginRight: 20}} role="button" onClick={onAddAnother}>
+            {'Add another'}
```
Comment (ethandaya) [inline pages/migrate.tsx]: doesn't need the curlies, just triggers an extra rerender every time the parent button ref changes

File: pages/migrate.tsx (hunk 3 above)
Comment (sonochristian) [reply pages/migrate.tsx]: it does with the breakline tags (single parent node within parentheses), won't need these if I get rid of those 👍

File: pages/migrate.tsx (hunk 4 above)
Comment (sonochristian) [reply pages/migrate.tsx]: haha to be honest I wasn't quite sure how to wire up a new CSS class so went with the easiest thing I could, perhaps this should be a flexbox?

Comment (sonochristian) [reply pages/migrate.tsx]: ah yep, I see the CSS imports now 👍

**Review by ethandaya — APPROVED** [review]

**Comment by sonochristian** [comment]
Screen Shot 2021-09-08 at 1 16 36 am

Updated styles

**Comment by sonochristian** [comment]
@ethandaya I don't have perms to merge this in so feel free to do that for me 👍

**Pull request merged**